editor/state.js

```diff
@@ -3,6 +3,7 @@
  */
 import optimist from 'redux-optimist';
 import { combineReducers, applyMiddleware, createStore } from 'redux';
+import ReduxThunk from 'redux-thunk';
```

Comment:
Why do we need redux thunk? We're using an alternative in Gutenberg https://github.com/aduth/refx